app/controllers/concerns/application_shared.rb

```diff
@@ -106,9 +106,10 @@ def scope_by_resource_id(base_scope, resource_id)
     return base_scope.none if resource_id.blank?
 
     if resource_id.is_a? String
-      # The class is parameterizable and the id is in '123-myname' format, extract the id from it
-      if base_scope.klass.respond_to?(:to_param) && resource_id.start_with?(/\d+-/)
```

Comment:
ActiveRecord::Base defines `to_param` (and we add our own implementation in https://github.com/theforeman/foreman/blob/b9c42d48ba4515c0ff1160d4d551ee5a0d7702c2/app/models/concerns/parameterizable.rb) so this check is not needed